web/src/api/typings/tasks.ts

```diff
@@ -12,11 +12,18 @@ export type Task = {
         name: string;
         id: number;
     };
-    user_id: string;
+    user: {
+        id: string;
+        name: string;
+    };
     is_validation: boolean;
     deadline: string;
 };
 
+export type Task = Omit<ApiTask, 'user'> & {
```

Comment:
What is the difference between entities Task and ApiTask?

Reply:
Task has user_id, ApiTask user:{id, name}